oioswift/proxy/controllers/obj.py

```diff
@@ -330,6 +363,11 @@ def PUT(self, req):
         headers = self._prepare_headers(req)
         with closing_if_possible(data_source):
             resp = self._store_object(req, data_source, headers)
+        if old_slo_manifest and resp.is_success:
+            self.app.logger.debug(
+                'Previous object %s was a SLO, deleting parts',
+                req.path)
```

Comment:
After checking https://developer.openstack.org/api-ref/object-store/index.html#copy-object : I've learned that  we may COPY only the manifest and have segments referenced several times, so if this features is used by customers, we will silently delete data !

Please add a note on Release Note

```To copy the manifest object, you include the multipart-manifest=get query string in the COPY request. The new object contains the same manifest as the original. The segment objects are not copied. Instead, both the original and new manifest objects share the same set of segment objects.```

Reply:
OK. This pull-request may be wrong. Now I think it's the client's responsibility to delete old segments (in my case: swift3).